### PR TITLE
Mila/count update proto

### PR DESCRIPTION
**Pull request by milaGGL**
Proto is updated based on the changes in this PR: https://github.com/firebase/firebase-android-sdk/pull/3847

### CODE REVIEW

File: packages/firestore/src/protos/google/firestore/v1/firestore.proto [hunk 1]
```diff
@@ -522,6 +545,41 @@ message RunQueryResponse {
...
+  // `projects/my-project/databases/my-database/documents` or
+  // `projects/my-project/databases/my-database/documents/chatrooms/my-chatroom`
+  string parent = 1 [(google.api.field_behavior) = REQUIRED];;
+
+  // The query to run.
+  oneof query_type {
+    // An aggregation query.
+    StructuredAggregationQuery structured_aggregation_query = 2;
+  }
+
```
Comment (milaGGL) [inline packages/firestore/src/protos/google/firestore/v1/firestore.proto:563]: the consistency_selector is not added to this proto

File: packages/firestore/src/protos/google/firestore/v1/aggregation_result.proto [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// query, unlike document queries which can have different fields present for
+// each result.
+message AggregationResult {
+  // The result of the aggregation functions, ex: `COUNT(*) AS total_docs`.
+  //
+  // The key is the [alias][google.firestore.v1.StructuredAggregationQuery.Aggregation.alias]
+  // assigned to the aggregation function on input and the size of this map
+  // equals the number of aggregation functions in the query.
+  map<string, Value> aggregate_fields = 2;
+}
```
Comment (dconeybe) [inline packages/firestore/src/protos/google/firestore/v1/aggregation_result.proto:42]: Please add a new line at the end of this file.

File: packages/firestore/src/protos/google/firestore/v1/firestore.proto [hunk 3]
```diff
@@ -17,6 +17,7 @@ syntax = "proto3";
 package google.firestore.v1;
 
 import "google/api/annotations.proto";
+import "google/firestore/v1/aggregation_result.proto";
```
Comment (dconeybe) [inline packages/firestore/src/protos/google/firestore/v1/firestore.proto:20]: nit: move this import of `aggregation_result.proto` into its alphabetically-sorted location.

**Review by dconeybe — CHANGES_REQUESTED** [review]
Great job! I've left a few minor comments to address.

File: packages/firestore/src/protos/google/firestore/v1/firestore.proto [hunk 4]
```diff
@@ -522,6 +545,62 @@ message RunQueryResponse {
...
 
+// The request for [Firestore.RunAggregationQuery][google.firestore.v1.Firestore.RunAggregationQuery].
+message RunAggregationQueryRequest {
+  // Required. The parent resource name. In the format:
+  // `projects/{project_id}/databases/{database_id}/documents` or
+  // `projects/{project_id}/databases/{database_id}/documents/{document_path}`.
+  // For example:
+  // `projects/my-project/databases/my-database/documents` or
+  // `projects/my-project/databases/my-database/documents/chatrooms/my-chatroom`
+  string parent = 1 [(google.api.field_behavior) = REQUIRED];
```
Comment (milaGGL) [inline packages/firestore/src/protos/google/firestore/v1/firestore.proto:556]: In andriod-sdk proto file, this line is stated as `string parent = 1; ` .
Other required field in web-sdk firestore.proto always trailed by `[(google.api.field_behavior) = REQUIRED]`.   So I added the same for this line.
Is this correct?

Comment (dconeybe) [reply packages/firestore/src/protos/google/firestore/v1/firestore.proto:556]: I think you are right to append the "REQUIRED" thing. It probably doesn't really matter. The proto in google3 does indeed have the REQUIRED clause: http://google3/google/firestore/v1beta1/firestore.proto;l=703;rcl=440401402. So it's probably good to have it there.

File: packages/firestore/src/protos/google/firestore/v1/firestore.proto [hunk 5]
```diff
@@ -133,6 +134,28 @@ service Firestore {
...
+  rpc RunAggregationQuery(RunAggregationQueryRequest) returns (stream RunAggregationQueryResponse) {
+    option (google.api.http) = {
+      post: "/v1/{parent=projects/*/databases/*/documents}:runAggregationQuery"
+      body: "*"
+      additional_bindings {
+        post: "/v1/{parent=projects/*/databases/*/documents/*/**}:runAggregationQuery"
+        body: "*"
+      }
+    };
+  }
```
Comment (dconeybe) [inline packages/firestore/src/protos/google/firestore/v1/firestore.proto:158]: nit: Add a blank line after this closing `}` too.

**Review by dconeybe — CHANGES_REQUESTED** [review]

**Review by dconeybe — APPROVED** [review]

**Pull request merged**